xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -989,6 +990,7 @@ static constexpr unsigned int SYSTEM_LOCALE = 1012;
 #define LISTITEM_PVR_INSTANCE_NAME  (LISTITEM_START + 218)
 #define LISTITEM_CHANNEL_LOGO       (LISTITEM_START + 219)
 #define LISTITEM_PVR_GROUP_ORIGIN   (LISTITEM_START + 220)
+#define LISTITEM_EPISODEPART        (LISTITEM_START + 221)
```

Comment:
This value (LISTITEM_START + 221) will collide with unmerged PR#24096.

https://github.com/xbmc/xbmc/blob/affc44682761fbcbc9243b840c91d6780be8301c/xbmc/guilib/guiinfo/GUIInfoLabels.h#L992C1-L992C61

Reply:
Whichever PR gets merged first, the other PR needs to be rebased to resolve the conflict. Nothing that needs to be charged in this PR as if now.